### PR TITLE
feature(ui-ux): add new header and wallet stacks v2

### PR DESCRIPTION
**Pull request by kyleleow**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
- To add new header title
- To add wallet stacks v2 that will uses the new header title

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/Main.web.tsx [hunk 1]
```diff
@@ -9,25 +9,28 @@ import { PlaygroundNavigator } from './PlaygroundNavigator/PlaygroundNavigator'
...
-      <View style={styles.phone}>
-        <RootNavigator />
-      </View>
-
-      {env.name !== EnvironmentName.Production && (
-        <View style={[styles.phone, tailwind('bg-white ml-2')]}>
-          <NavigationContainer theme={DeFiChainTheme}>
-            <PlaygroundNavigator />
-          </NavigationContainer>
+    <SafeAreaProvider>
```
Comment (thedoublejay) [inline mobile-app/app/screens/Main.web.tsx:19]: didn't we have this already somewhere?

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: mobile-app/app/constants/Theme.ts [hunk 2]
```diff
@@ -7,9 +7,8 @@ export function getDefaultTheme (isLight: boolean): Theme {
     ...defaultTheme,
     colors: {
       ...defaultTheme.colors,
-      primary: isLight ? '#ff00af' : '#EE2CB1',
```
Comment (thedoublejay) [inline mobile-app/app/constants/Theme.ts:10]: wouldn't this cause regression issues?

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/Main.web.tsx (hunk 1 above)
Comment (kyleleow) [reply mobile-app/app/screens/Main.web.tsx:19]: We have it on `Main.tsx` but not on web

File: mobile-app/app/constants/Theme.ts (hunk 2 above)
Comment (kyleleow) [reply mobile-app/app/constants/Theme.ts:10]: updated

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**